[2025-03-28T02:14:51.108+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-28T02:14:51.875+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_dbt_clean_reviews_pipeline.dbt_run_transactional scheduled__2024-03-18T00:00:00+00:00 [queued]>
[2025-03-28T02:14:52.027+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_dbt_clean_reviews_pipeline.dbt_run_transactional scheduled__2024-03-18T00:00:00+00:00 [queued]>
[2025-03-28T02:14:52.028+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-28T02:14:52.464+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run_transactional> on 2024-03-18 00:00:00+00:00
[2025-03-28T02:14:52.557+0000] {standard_task_runner.py:72} INFO - Started process 187152 to run task
[2025-03-28T02:14:52.937+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'load_dbt_clean_reviews_pipeline', 'dbt_run_transactional', 'scheduled__2024-03-18T00:00:00+00:00', '--job-id', '1992', '--raw', '--subdir', 'DAGS_FOLDER/dag_load_reviews.py', '--cfg-path', '/tmp/tmpdjfaldcy']
[2025-03-28T02:14:52.963+0000] {standard_task_runner.py:105} INFO - Job 1992: Subtask dbt_run_transactional
[2025-03-28T02:14:56.454+0000] {task_command.py:467} INFO - Running <TaskInstance: load_dbt_clean_reviews_pipeline.dbt_run_transactional scheduled__2024-03-18T00:00:00+00:00 [running]> on host DESKTOP-IM7V4RI.
[2025-03-28T02:15:01.681+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohammed' AIRFLOW_CTX_DAG_ID='load_dbt_clean_reviews_pipeline' AIRFLOW_CTX_TASK_ID='dbt_run_transactional' AIRFLOW_CTX_EXECUTION_DATE='2024-03-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-18T00:00:00+00:00'
[2025-03-28T02:15:01.712+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-28T02:15:01.713+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-28T02:15:01.714+0000] {logging_mixin.py:190} INFO - Current task name:dbt_run_transactional state:running start_date:2025-03-28 02:14:51.879035+00:00
[2025-03-28T02:15:01.715+0000] {logging_mixin.py:190} INFO - Dag name:load_dbt_clean_reviews_pipeline and current dag run status:running
[2025-03-28T02:15:01.717+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-28T02:15:01.775+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-28T02:15:01.888+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /home/mohammed/.dbt/dbt_projects/bank_reviews_transactional && dbt run']
[2025-03-28T02:15:02.019+0000] {subprocess.py:99} INFO - Output:
[2025-03-28T02:15:50.482+0000] {job.py:229} INFO - Heartbeat recovered after 21.83 seconds
[2025-03-28T02:15:58.786+0000] {job.py:229} INFO - Heartbeat recovered after 24.53 seconds
[2025-03-28T02:16:02.767+0000] {subprocess.py:106} INFO - [0m02:16:02  Running with dbt=1.9.3
[2025-03-28T02:16:09.636+0000] {subprocess.py:106} INFO - [0m02:16:09  Registered adapter: postgres=1.9.0
[2025-03-28T02:16:13.677+0000] {subprocess.py:106} INFO - [0m02:16:13  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-28T02:16:13.678+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-03-28T02:16:13.679+0000] {subprocess.py:106} INFO - - models.bank_reviews_transactional.transactional
[2025-03-28T02:16:16.322+0000] {subprocess.py:106} INFO - [0m02:16:16  Found 1 model, 1 source, 433 macros
[2025-03-28T02:16:16.363+0000] {subprocess.py:106} INFO - [0m02:16:16
[2025-03-28T02:16:16.400+0000] {subprocess.py:106} INFO - [0m02:16:16  Concurrency: 4 threads (target='dev')
[2025-03-28T02:16:16.411+0000] {subprocess.py:106} INFO - [0m02:16:16
[2025-03-28T02:16:18.251+0000] {subprocess.py:106} INFO - [0m02:16:18  1 of 1 START sql table model transactional.clean_reviews ....................... [RUN]
[2025-03-28T02:16:42.926+0000] {job.py:229} INFO - Heartbeat recovered after 27.93 seconds
[2025-03-28T02:17:11.012+0000] {job.py:229} INFO - Heartbeat recovered after 28.11 seconds
[2025-03-28T02:17:13.549+0000] {subprocess.py:106} INFO - [0m02:17:13  1 of 1 OK created sql table model transactional.clean_reviews .................. [[32mSELECT 153[0m in 55.24s]
[2025-03-28T02:17:13.795+0000] {subprocess.py:106} INFO - [0m02:17:13
[2025-03-28T02:17:13.829+0000] {subprocess.py:106} INFO - [0m02:17:13  Finished running 1 table model in 0 hours 0 minutes and 57.38 seconds (57.38s).
[2025-03-28T02:17:14.378+0000] {subprocess.py:106} INFO - [0m02:17:14
[2025-03-28T02:17:14.386+0000] {subprocess.py:106} INFO - [0m02:17:14  [32mCompleted successfully[0m
[2025-03-28T02:17:14.394+0000] {subprocess.py:106} INFO - [0m02:17:14
[2025-03-28T02:17:14.413+0000] {subprocess.py:106} INFO - [0m02:17:14  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-28T02:17:20.584+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-28T02:17:27.694+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-28T02:17:27.734+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=load_dbt_clean_reviews_pipeline, task_id=dbt_run_transactional, run_id=scheduled__2024-03-18T00:00:00+00:00, execution_date=20240318T000000, start_date=20250328T021451, end_date=20250328T021727
[2025-03-28T02:17:28.549+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-28T02:17:28.551+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-28T02:17:28.552+0000] {logging_mixin.py:190} INFO - Dag name:load_dbt_clean_reviews_pipeline queued_at:2025-03-28 02:14:46.241698+00:00
[2025-03-28T02:17:28.554+0000] {logging_mixin.py:190} INFO - Task hostname:DESKTOP-IM7V4RI. operator:BashOperator
[2025-03-28T02:17:28.970+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-28T02:17:29.318+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
